### PR TITLE
fix(accordion): respect presence of expanded in html

### PR DESCRIPTION
**Pull request by bennypowers**
## What I did
test and ensure that when `expanded` is present on an accordion header in HTML, it's associated panel will expand on the page.

## Testing instructions
unit tests, check the DP, play with the demos

### CODE REVIEW

File: elements/pfe-accordion/test/pfe-accordion.spec.ts [hunk 1]
```diff
@@ -909,6 +909,63 @@ describe('<pfe-accordion>', function() {
...
+    it('expands the second panel', function() {
+      expect(secondPanel).to.have.attribute('expanded');
+    });
+    it('hides the third panel', function() {
+      expect(panels[2]).to.not.have.attribute('expanded');
+    });
+  });
+
+  describe('with single attribute and expanded attribute on two headers', function() {
+    beforeEach(async function() {
```
[the remark this answers is not in the document]
Comment (bennypowers) [reply elements/pfe-accordion/test/pfe-accordion.spec.ts:942]: added. thanks

[the remark this answers is not in the document]
Comment (bennypowers) [reply elements/pfe-accordion/test/pfe-accordion.spec.ts:942]: for `single`, I'm assuming that the last `expanded` panel wins

[the remark this answers is not in the document]
Comment (brianferry) [reply elements/pfe-accordion/test/pfe-accordion.spec.ts:942]: So our test on 960 would need to be updated too to account for that, right?  Only the second panel should be expanded then

[the remark this answers is not in the document]
Comment (bennypowers) [reply elements/pfe-accordion/test/pfe-accordion.spec.ts:942]: gotcha! thank you for the careful review

**Review by brianferry — CHANGES_REQUESTED** [review]
Looks good to me code / testing wise, resolve when all testing is completed

**Review by nikkimk — APPROVED** [review]

**Review by brianferry — APPROVED** [review]

**Pull request merged**